core/unit_test/CMakeLists.txt

```diff
@@ -827,11 +827,22 @@ endif()
 
 # FIXME_OPENMPTARGET, FIXME_OPENACC - Comment non-passing tests with the NVIDIA HPC compiler nvc++
 if ((KOKKOS_ENABLE_OPENMPTARGET OR KOKKOS_ENABLE_OPENACC) AND KOKKOS_CXX_COMPILER_ID STREQUAL NVHPC)
-  SET(DEFAULT_DEVICE_SOURCES
-    UnitTestMainInit.cpp
-    TestInitializationSettings.cpp
-    TestParseCmdLineArgsAndEnvVars.cpp
-    default/TestDefaultDeviceType.cpp
+  LIST(REMOVE_ITEM DEFAULT_DEVICE_SOURCES
+    TestSharedSpace.cpp
```

Comment:
Duplicate deletes: `TestSharedSpace.cpp` and `TestSharedHostPinnedSpace.cpp`

Reply:
Seyong is pointing out https://github.com/kokkos/kokkos/pull/6260/files#diff-3b1b58692f137b8d5da01ac74cce73ee0be6d04fdac5ad0c2bca1bb513e642fbR818-R826